myapp/api/management/commands/createsuperuser.py

```diff
@@ -12,15 +12,15 @@ class Command(BaseCommand):
 
     def handle(self, *args, **kwargs):
         email = input("Enter email: ")
-        preferred_name = input("Enter first name: ")
+        first_name = input("Enter first name: ")
```

Comment:
where did this change come from?

Reply:
Creating super user requires first name which is not correctly assigned in the code, Ryan Chen said he forgot to modify the code when working with preferred name, hence the change.

Reply:
Ah I see